specify/specify7
# Add isLoanable only for loans when choosing record set

Fixes #5217 

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Take note of your prep types and which ones are loanable or not 
- Create a loan interaction 
- Enter the cat number of a CO that has some prep with prep type loanable and some not loanable (or a record set that contains such CO) 
- [ ] verify that only the prep with prep type loanable are proposed 
-  Proceed with the same flow but for another interaction, gift by example 
- [ ] verify that the prep with prep type not loanable are presented as option
